src/renderer/pages/PreferencesPage.js

```diff
@@ -133,6 +153,9 @@ class PreferencesPage extends React.Component {
         <PreferencesSection title='Default torrent app'>
           {this.setDefaultAppButton()}
         </PreferencesSection>
+        <PreferencesSection title='Telemetry'>
```

Comment:
I would title this 'Privacy' instead of 'Telemetry' since most users won't know what the latter means.